inference-engine/.clang-format

```diff
@@ -0,0 +1,5 @@
+BasedOnStyle: Google
```

Comment:
All those "." files likely should be in the root of the repo. For example, if we add some more C++ in a separate folder we would reuse those formatting rules.

Reply:
Sure. I just kept them here since this was the only directory that had C++ code in it like this, but I'm happy to move these to the root-level directory.